packages/client.api/src/index.ts

```diff
@@ -119,14 +117,14 @@ export type {
   QueryReturnType,
   QuerySignatureFromDef,
 } from "./queries/Queries.js";
+export { TimeseriesDurationMapping } from "./timeseries/timeseries.js";
 export type {
   TimeSeriesPoint,
   TimeSeriesProperty,
   TimeSeriesQuery,
 } from "./timeseries/timeseries.js";
-export { TimeseriesDurationMapping } from "./timeseries/timeseries.js";
-
 export type { LinkedType, LinkNames } from "./util/LinkUtils.js";
 export type { NOOP } from "./util/NOOP.js";
+export type { PartialBy } from "./util/PartialBy.js";
```

Comment:
Exposed for @osdk/client. I think this actually goes away once I do the same treatment to queries that I just did to actions.